### ISSUE TITLE
Update year everywhere to be 2019

### ISSUE THREAD

**Issue by desilinguist** [EducationalTestingService/skll#447]
Documentation, code, etc.

**Comment by SupreethBaliga** [comment]
Hi. I would like to take this up. Can you please give me more details about the files and other related stuff?
Thanks.

**Comment by desilinguist** [comment]
Hi @SupreethBaliga! Thanks for volunteering for this issue. Happy to have your help! Looks like the only file that needs updating is `doc/conf.py` (for the `copyright` and `epub_copyright` strings).

**Comment by SupreethBaliga** [comment]
The issue looks a bit unclear. Please help me with this. You basically want me to just change all years to 2019. Even the copyright stuff?

**Comment by desilinguist** [comment]
No, you just need to convert the following two strings in `conf.py`L

```
copyright = u'2012-2017, Educational Testing Service'
epub_copyright = u'2012-2014, Educational Testing Service'
```

Both those strings should read "2012-2019, Educational Testing Service".

**Comment by desilinguist** [comment]
Addressed by #456.

**Comment by desilinguist** [comment]
We forgot to update LICENSE.txt`.

**Comment by desilinguist** [comment]
Addressed by #568
